### PR TITLE
Prepare release 0.7.16

### PR DESCRIPTION
**Pull request by DoumanAsh**
@fafhrd91 Would we like to not re-export full `actix` at all?

### CODE REVIEW

File: src/extractor.rs [hunk 1]
```diff
@@ -764,6 +764,9 @@ impl Default for EitherCollisionStrategy {
     }
 }
 
+///Determines Either extractor configuration
+///
+///By default uses default `EitherCollisionStrategy`
```
Comment (axos88) [inline src/extractor.rs]:
```suggestion
///By default uses default `EitherCollisionStrategy::FastestSuccessful`
```

Comment (DoumanAsh) [reply src/extractor.rs]: Oh damn, I forgot to add variant itself, thx for noticing it

File: src/extractor.rs [hunk 2]
```diff
@@ -764,6 +763,9 @@ impl Default for EitherCollisionStrategy {
     }
 }
 
+///Determines Either extractor configuration
+///
+///By default uses default `EitherCollisionStrategy::FastestSuccessful`
```
Comment (axos88) [inline src/extractor.rs]:
```suggestion
///By default uses `EitherCollisionStrategy::FastestSuccessful`
```

Comment (axos88) [reply src/extractor.rs]: Sorry, just noticed this now :/

**Pull request merged**